### PR TITLE
prv_getParameters / fix query len check

### PR DESCRIPTION
**Pull request by semhoun**
When the query is QUERY_QUEUE_MODE test of the len is compared with QUERY_BINDING_LEN and not QUERY_QUEUE_MODE_LEN.
It's like an erronous copy/paste.

### CODE REVIEW

**Comment by rettichschnidi** [comment]
Thanks for your PR. Before we can integrate, please sign the [ECA](https://accounts.eclipse.org/user/eca).

**Comment by semhoun** [comment]
I signed the ECA.

**Comment by rettichschnidi** [comment]
CI requires a commit to have a body. If you annotate the commit with what you described in the PR, CI will pass.

File: core/registration.c
Comment (semhoun) [inline core/registration.c:1]: When the query is QUERY_QUEUE_MODE test of the len is compared with QUERY_BINDING_LEN and not QUERY_QUEUE_MODE_LEN.
It's like an erronous copy/paste.

**Review by semhoun — COMMENTED** [review]
When the query is QUERY_QUEUE_MODE test of the len is compared with QUERY_BINDING_LEN and not QUERY_QUEUE_MODE_LEN.
It's like an erronous copy/paste.

**Review by semhoun — COMMENTED** [review]
Sorry, i don't know how to annotate a commit after pushin it.

**Comment by rettichschnidi** [comment]
Sorry, amend is what I meant (`git commit --amend`). You can read up on how to do this [here](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History).

Since you allowed maintainer edits, I just massaged the commit a bit, so it (hopefully) passes the CI. Will merge once it does.

Thanks a lot!

**Review by rettichschnidi — APPROVED** [review]

**Comment by semhoun** [comment]
Thanks, i will know for the future.

**Pull request merged**
